### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by AvshRLev**

### CODE REVIEW

File: static/ui.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function createChildDependeciesListItems(queryPackage, parentID) {
   if (parent.children.length > 0) {
     return;
   }
-  fetch("http://localhost:5000/" + queryPackage)
+  fetch("http://127.0.0.1:5000/" + queryPackage)
```
Comment (buzzdan) [inline static/ui.js]: dont use any base url here.\
your browser can detect it if you just start your path from `/`:
```js
fetch("/" + queryPackage)
```

File: main.py [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 import os
 
-redis_host = 'localhost'
+load_dotenv()
+redis_host = os.environ['redis']
 redis_port = 6379
 redis_client = redis.StrictRedis(host=redis_host, port=redis_port, decode_responses=True)
 
-npm_base_url = 'https://registry.npmjs.org/'
+npm_base_url = os.environ['npm_base_url']
```
Comment (buzzdan) [inline main.py:14]: you should have defaults somewhere.\
where are those located ?

File: static/ui.js (hunk 1 above)
Comment (AvshRLev) [reply static/ui.js]: amazing!

File: main.py (hunk 2 above)
Comment (AvshRLev) [reply main.py:14]: .env file

**Pull request merged**